src/main/steps/applicant1/changing-to-a-sole-application/template.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "common/page.njk" %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: phase
+    },
+    html: feedback + languageToggle
+  }) }}
+{% endblock %}
+
+{% block page_content %}
+  <h1 class="govuk-heading-l">{{ title }}</h1>
+  <p class="govuk-body">{{ line1 }}</p>
```

Comment:
this also seems to be the same as switch to sole, maybe we can have an if statement for the line1 and include that file? unless theres other differences.